src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,45 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import java.util.Random;
+
 public class HelloWorld {
+    public static void main(String[] args) {
+
+        Random random = new Random();
+
+        Figure[] figures = new Figure[random.nextInt(10)];
+
+        for (int i = 0; i < figures.length; i++) {
+            switch (random.nextInt(5)) {
```

Comment:
declare 5 as a constant, now it's magic number
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names